### ISSUE TITLE
Locale update for 1.1

### ISSUE THREAD

**Issue by tobias-klein** [ezra-bible-app/ezra-bible-app#363]
These are the updated texts for 1.1.

**Various sections:**
* app-language
* detect-locale
* select-chapter-before-loading
* open-complete-book
* open-chapters-large-books
* open-chapters-all-books

**Emoji section:**
```
  "emoji": {
    "search": "Search emojis...",
    "notFound": "No emojis found",
    "categories": {
      "recents": "Recent Emojis",
      "smileys": "Smileys & Emotion",
      "people": "People & Body",
      "animals": "Animals & Nature",
      "food": "Food & Drink",
      "activities": "Activities",
      "travel": "Travel & Places",
      "objects": "Objects",
      "symbols": "Symbols",
      "flags": "Flags"
    }
  },
```

**Shortcuts section:**
```
  "shortcuts": {
    "tab-title": "Keyboard Shortcuts",
    "description": "You can use the following shortcuts:",

    "summary" : {
      "font-section-title": "Font",
      "current-tab-search-section-title": "Current Tab Search",
      "miscellaneous-section-title": "Miscellaneous",
      "in-tab-search": "Open current tab search",
      "hide-in-tab-search-form": "Hide current tab search form",
      "jump-to-next-search-occurance": "Jump to next search occurance",
      "jump-to-previous-occurance": "Jump to previous occurance",
      "copy-selected-verses-to-clipboard": "Copy selected verses to clipboard",
      "toggle-fullscreen-only-Win-Linux": "Toggle fullscreen (only Win/Linux)",
      "enable-dynamic-strongs-display": "Enable dynamic Strong's display",
      "open-new-tab": "Open new tab",
      "increase-current-font-size": "Increase current font size",
      "decrease-current-font-size": "Decrease current font size",
      "reset-font-size-to-default": "Reset font size to default"
    },

    "shortcut": {
      "in-tab-search":"Ctrl/Cmd + f",
      "hide-in-tab-search-form": "Esc",
      "jump-to-next-search-occurance": "Enter",
      "jump-to-previous-occurance": "Shift + Enter",
      "copy-selected-verses-to-clipboard": "Ctrl/Cmd + c",
      "toggle-fullscreen-only-Win-Linux": "F11",
      "enable-dynamic-strongs-display": "Shift + mouse over verse",
      "open-new-tab": "Ctrl/Cmd + t",
      "increase-current-font-size": "Ctrl/Cmd +",
      "decrease-current-font-size": "Ctrl/Cmd -",
      "reset-font-size-to-default": "Ctrl/Cmd 0"
    }
  }
```

**Cordova section:**
* starting-app
* init-i18n
* init-sword
* init-database
* init-user-interface
* loading-settings
* waiting-app-ready

@lemtom Could you submit a pull request for the French and Dutch translation.
@MartinIIOT Could you submit a pull request for the Slovakian translation.
@reyespinosa1996 Could you submit a pull request for the Spanish translation.
@augustin-colesnic Could you submit a pull request for the Romanian translation.
@zhuiks Could you submit pull requests for the Ukrainian and Russian translation.

I am planning to release 1.1.0 on June 26th. Thanks for your help!

**Comment by augustin-colesnic** [comment]
@tobias-klein  
### Romanian is all ok with the above.
No changes needs to be done for Romanian.

<img width="460" alt="Screenshot 2021-06-21 at 10 38 07" src="https://user-images.githubusercontent.com/592416/122724760-eeda6580-d27c-11eb-83a4-8601f8fd453c.png">
<img width="833" alt="Screenshot 2021-06-21 at 10 38 31" src="https://user-images.githubusercontent.com/592416/122724768-f00b9280-d27c-11eb-91ee-bb9ceb9747d0.png">

**Comment by augustin-colesnic** [comment]
@tobias-klein  I noticed that Repositories and Languages 'buttons'  do not pick up the translations (e.g. - romanian and spanish in the attached screenshots)  although they are translated.

<img width="562" alt="Screenshot 2021-06-21 at 11 36 08" src="https://user-images.githubusercontent.com/592416/122733029-7926c780-d285-11eb-87ab-7a25f5b5da7a.png">

<img width="700" alt="Screenshot 2021-06-21 at 11 39 09" src="https://user-images.githubusercontent.com/592416/122733034-7926c780-d285-11eb-84ba-a4aa1a1fc0bf.png">

<img width="1089" alt="Screenshot 2021-06-21 at 11 34 49" src="https://user-images.githubusercontent.com/592416/122733022-775d0400-d285-11eb-9824-4f136d4d6612.png">

**Comment by tobias-klein** [comment]
> @tobias-klein I noticed that step sections (1. 2. and 4.) do not pick up the translations (e.g. - Romanian and Spanish in the attached screenshots) although they are translated.

Thanks for reporting this, @augustin-colesnic. That's a bug. I am going to have a look.

**Comment by tobias-klein** [comment]
@augustin-colesnic It's a bit off-topic for this issue, but anyway ... I fixed the bug. Can you have another look?

**Comment by tobias-klein** [comment]
> [ ] Some of them from **bible-browser** section I do NOT understand exactly what their meaning is:
> "open-complete-book": "**Open books completely**", would this be _Open the finished books_?
No ... This means that all books are always opened with all chapters at once. So this corresponds to the existing behavior in 1.0.0.

> "open-chapters-large-books": "**Open large books chapter-wise**", would this be _Open long books based on chapter_?
Yes.

> "open-chapters-all-books": "**Open all books chapter-wise**" would this be _Open all books based on chapter_?
Yes.

**Comment by augustin-colesnic** [comment]
> @augustin-colesnic It's a bit off-topic for this issue, but anyway ... I fixed the bug. Can you have another look?

**Issue fixed.** 
<img width="412" alt="Screenshot 2021-06-21 at 19 59 53" src="https://user-images.githubusercontent.com/592416/122800344-7cdd3d00-d2cb-11eb-9ad7-f83eb6ce95da.png">

**Comment by tobias-klein** [comment]
@zhuiks My latest change broke the test. What I did is close the options menu immediately instead of waiting for the locale change to happen. Maybe it's enough to insert a small delay? Would you mind taking a look?

**Comment by reyespinosa1996** [comment]
The new behavior of chapters opening based on books weight should be accompanied by a disclaimer (Less memory usage and better memory performance). This is because it is challenging to give the "chapter-wise" concept when translating to Spanish.

**Comment by reyespinosa1996** [comment]
I know this is a bit off-topic, but...
>       "enable-dynamic-strongs-display": "Shift + mouse over verse",
I think "cursor over verse"  should be better. Because there is people using touchpads.

**Comment by zhuiks** [comment]
> The new behavior of chapters opening based on books weight should be accompanied by a disclaimer (Less memory usage and better memory performance). This is because it is challenging to give the "chapter-wise" concept when translating to Spanish.

I agree, it's a very hard concept to translate within the small step. Ukrainian comes out a bit wordy, but I'm still not sure if the concept would be clear. Which I think should be the main goal of the translation.

![Знімок екрану з 2021-06-22 14-29-26](https://user-images.githubusercontent.com/3584119/122988632-916f1300-d367-11eb-925e-52a9057f3e13.png)

@tobias-klein maybe we can drop this option in the future once we implement #341

**Comment by zhuiks** [comment]
> I know this is a bit off-topic, but...
> 
> > ```
> >   "enable-dynamic-strongs-display": "Shift + mouse over verse",
> > ```
> 
> I think "cursor over verse" should be better. Because there is people using touchpads.

Unfortunately, this one works only in the desktop app where both keyboard and mouse are available

**Comment by reyespinosa1996** [comment]
> > I know this is a bit off-topic, but...
> > > ```
> > >   "enable-dynamic-strongs-display": "Shift + mouse over verse",
> > > ```
> > 
> > 
> > I think "cursor over verse" should be better. Because there is people using touchpads.
> 
> Unfortunately, this one works only in the desktop app where both keyboard and mouse are available
In android and/or iOS people use mouse?

**Comment by tobias-klein** [comment]
> @tobias-klein maybe we can drop this option in the future once we implement #341

Yes, we can consider that. Or alternatively, with a different options menu design we could add an explanatory sentence in the future. At some point we will need to redesign the options menu as we keep adding more options in the future.

**Comment by tobias-klein** [comment]
> I think "cursor over verse" should be better. Because there is people using touchpads.

Mmmm ... yeah, good point. I'll reconsider this and get back once more.

**Comment by tobias-klein** [comment]
We're done with most languages now.

@MartinIIOT Reminder to submit a pull request for the Slovakian translation. I am planning to release on Saturday.
